README.md

```diff
@@ -521,6 +521,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [OkHttp](http://square.github.io/okhttp/) - HTTP+SPDY client.
 * [Undertow](http://undertow.io/) - Web server providing both blocking and non-blocking API’s based on NIO. Used as a network layer in WildFly.
 * [urnlib](https://github.com/slub/urnlib) - Java library for representing, parsing and encoding URNs as in RFC 2141.
+* [MINA](https://mina.apache.org/) - Apache MINA is a network application framework which helps users develop high performance and high scalability network applications easily. It provides an abstract event-driven asynchronous API over various transports such as TCP/IP and UDP/IP via Java NIO.
```

Comment:
suggest: "Abstract event-driven async I/O API for network operations over TCP/IP and UDP/IP via Java NIO."